tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -40,8 +41,9 @@
 FILES_KEY_KEY = "key"
 
 OUTPUT_JOB_ID_KEY = "jobId"
+OUTPUT_ORCA_ARCHIVE_LOCATION_KEY = "orcaArchiveLocation"
 
-LOGGER = CumulusLogger()
+LOGGER = CumulusLogger(name="Orca")
```

Comment:
I think everywhere else Orca is all lower case for logging. Maybe change to `name="orca"`

Reply:
Nope. Standard is "Orca". Looks like shared_db uses the wrong one, so I'll get that swapped over.

Reply:
Changed